scout/server/blueprints/login/views.py

```diff
@@ -106,7 +106,6 @@ def logout():
     for provider in ["GOOGLE", "KEYCLOAK"]:
         if current_app.config.get(provider):
             controllers.logout_oidc_user(session, provider)
-    session.clear()
```

Comment:
If I remember correctly I've added this line in my keycloack, PR! Well spotted!

Reply:
That would make a lot of sense - surely we should have noticed if it was from way back when?!